v22.2/common-table-expressions.md

```diff
@@ -295,7 +295,155 @@ SELECT * FROM cte LIMIT 10;
 (10 rows)
 ~~~
 
-While this practice works for testing and debugging, Cockroach Labs does not recommend it in production.
+While adding a limit to prevent infinite recursion works for testing and debugging, Cockroach Labs does not recommend it in production. It is best practice to ensure that recursive subqueries have an explicit end condition.
+
+### Loose index scan using a recursive CTE
+
```

Comment:
General nit: (similar to the PR I just reviewed on trigram indexes) some of the lead-ins to the SQL are a little abrupt. The intro here to this example is mainly telling me what it is, rather than how the example will flow/accomplish. It could be beneficial to add a sentence underneath this to only mention that the example will first scan an index without recursive CTE and then use recursive CTE.